Python/ds/Queue.py

```diff
@@ -0,0 +1,32 @@
+# Stack implementation using List in Python
+
+class Queue :
+  def __init__(self):
+    self.items =[]
+    
+  def print_queue(self):
+    print "\nQueue: ", self.items
+    print "Length of Queue: ", len(self.items), "\n"
+      
+  def enqueue(self, item):
+    self.items.insert(0,item)
+    print "Enqueued value: ", item
+  
+  def dequeue(self):
+    print "Dequeued value: ",self.items.pop()
+  
```

Comment:
Remove this extra line.

Reply:
It would be even better if you could add a function to print the top most element.

Reply:
Thanks for the suggestions 🙂